core/iwasm/fast-jit/jit_ir.c

```diff
@@ -1558,3 +1558,43 @@ _jit_insn_check_opnd_access_LookupSwitch(const JitInsn *insn)
     unsigned opcode = insn->opcode;
     return (insn_opnd_kind[opcode] == JIT_OPND_KIND_LookupSwitch);
 }
+
+#if defined(BUILD_TARGET_X86_64) || defined(BUILD_TARGET_AMD_64)
```

Comment:
How about not adding the macro control? Just to make it a global function, so that other targets can use it, e.g. x86_32 in the future.